lib/formats/pkcs8.js

```diff
@@ -624,8 +631,13 @@ function writePkcs8EdDSAPublic(key, der) {
 function writePkcs8EdDSAPrivate(key, der) {
 	der.endSequence();
 
-	var k = utils.mpNormalize(key.part.k.data, true);
 	der.startSequence(asn1.Ber.OctetString);
+	var k = utils.mpNormalize(key.part.k.data);
+	/* RFCs call for storing exactly 32 bytes, so strip any leading zeros */
+	while (k.length > 32 && k[0] == 0x00)
```

Comment:
jsl is complaining that this should be `===` not `==`. If that's fixed, `make check` will pass.

<https://jenkins.tritondatacenter.com/job/TritonDataCenter/job/node-sshpk/view/change-requests/job/PR-83/1/console>